### ISSUE TITLE
Dedicated Hamburger Menu for one option

### ISSUE THREAD

**Issue by Schweeeeeeeeeeeeeeee** [modrinth/code#2233]
### Please confirm the following.

- [X] I checked the [existing issues](https://github.com/modrinth/code/issues) for duplicate problems
- [X] I have tried resolving the issue using the [support portal](https://support.modrinth.com)

### Describe the bug
When modrinth plus is bought:
![image](https://github.com/user-attachments/assets/99230dea-a6f5-4dd4-a5eb-f556b87a4a01)
Without modrinth plus:
![image](https://github.com/user-attachments/assets/bc739a3c-3d69-4708-a782-bd820b11b948)

Theres a dedicated hamburger menu for one button if you bought modrinth+ because one of the 2 options dissapears

### Steps to reproduce

1. Buy Modrinth=
2. Click on More

### Expected behavior

Hamburgermenu turns into a download button when modrinth plus is bought

**Comment by brawaru** [comment]
It would probably make sense to just change the text instead, e.g. ‘About Modrinth+’.

**Comment by NullyIsHere** [comment]
> It would probably make sense to just change the text instead, e.g. ‘About Modrinth+’.

Yeah. That way you can look at what you have.

**Comment by Minenash** [comment]
> Hamburgermenu turns into a download button when modrinth plus is bought

I disagree. Not only would it being back "advertising on the main nav), but the full text is noticable longer. I think it might *barely* fit on your screen, but on any windows that's more narrow, it wouldn't fit

**Comment by Minenash** [comment]
> Yeah. That way you can look at what you have.

You already can in settings

**Comment by NullyIsHere** [comment]
Then just remove hamburger

**Comment by Erb3** [comment]
I believe this is no longer applicable.

**Comment by Schweeeeeeeeeeeeeeee** [comment]
Jup think so too
